src/components/App.js

```diff
@@ -1,16 +1,105 @@
 import React, { Component } from 'react';
-import logo from '../logo.svg';
 import '../styles/App.css';
 import Header from './Header.js';
 import Main from './Main.js';
-import Table from './Table.js';
+
+const USER = {
```

Comment:
Here's the dummy data! Keeping all of the data defined in App.js means that once we have everything working, it will be easy to swap it out with live data from a database.